docs/deployment/nexodus-service.md

```diff
@@ -38,12 +38,39 @@ The Makefile will install the https certs. You can view the cert in the Nexodus
 cat .certs/rootCA.pem
 ```
 
-In order to join a self-signed Nexodus Service from a remote node or view the Nexodus UI in your dev environment, you will need to install the cert on the remote machine. This is only necessary when the service is self-signed with a domain like we are using with the try.nexodus.127.0.0.1.nip.io domain for development.
+You can recreate that file at any time with the following.
+
+```console
+make cacerts
+```
+
+In order to join a self-signed Nexodus Service from a remote node or view the Nexodus UI in your dev environment, you will need to install the cert on the remote machine. This is only necessary when the service is self-signed with a domain like we are using with the `try.nexodus.127.0.0.1.nip.io` domain for development.
+
+Add the following host entries to `/etc/hosts` pointing to the IP the kind stack is running on.
+
+```console
+<IP of machine running the KIND stack> auth.try.nexodus.127.0.0.1.nip.io api.try.nexodus.127.0.0.1.nip.io try.nexodus.127.0.0.1.nip.io
+```
 
 Install [`mkcert`](https://github.com/FiloSottile/mkcert) on the agent node, copy the cert from the service running kind (`.certs/rootCA.pem`) to the remote node you will be joining (or viewing the web UI) and run the following.
 
 ```console
 CAROOT=$(pwd)/.certs mkcert -install
 ```
 
+Verify the service by attaching a node using built-in accounts as part of the kubernetes dev overlay build `make run-on-kind` provides.
+
+```console
+# from the nexodus repo directory root:
+make dist/nexd-linux-amd64
+sudo NEXD_LOGLEVEL=debug dist/nexd-linux-amd64 --username admin --password floofykittens  https://try.nexodus.127.0.0.1.nip.io
+```
```

Comment:
I meant to put this last time, but another option is running `make run-nexd-container`. It's an easy way to run one or more sandboxed nexd instances.

Reply:
Neat! I didn't know this was there.